tutorials/eda_contents/numpy/numpy.ipynb

```diff
@@ -0,0 +1,1408 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Disclaimer: Kindly be aware that the questions and datasets featured in this tutorial were originally presented by [Ryan Abernathy in \"An Introduction to Earth and Environmental Data Science\"](https://earth-env-data-science.github.io/intro.html)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Numpy and Matplotlib\n",
+    "\n",
+    "\n",
+    "#### Numpy\n",
+    "As you may already know, NumPy stands as a foundational Python package crucial for scientific computing. This library operates on ndarray objects, enabling swift and efficient calculations on arrays and matrices. Despite being utilized within Python, its rapid computing capabilities are attributed to components written in C or C++. Proficiency in this package is essential for a majority of computational tasks involving geoscience data. This workbook aims to illustrate fundamental applications of NumPy within this domain.\n",
```

Comment:
> As you may already know...

This reads strangely to me. It's a bit conversational for a tutorial. I'd just start this with "NumPy stands ..."

Reply:
updated